setup.py

```diff
@@ -13,14 +13,23 @@
     long_description=(open("README.rst").read() + "\n" +
                       open("CHANGES.rst").read()),
     classifiers=[
-        'Environment :: Web Environment',
-        'Framework :: Zope :: 4',
-        'License :: OSI Approved :: Zope Public License',
-        'Operating System :: OS Independent',
-        'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
-        'Topic :: Internet :: WWW/HTTP :: Site Management',
+        'Development Status :: 6 - Mature',
+        "Environment :: Web Environment",
+        "Framework :: Zope :: 4",
+        "Intended Audience :: Developers",
+        "License :: OSI Approved :: Zope Public License",
+        "Operating System :: OS Independent",
+        "Programming Language :: Python :: 2",
+        "Programming Language :: Python :: 2.7",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.5",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
```

Comment:
Python 3.8 is neither tested on TravisCI nor via `tox.ini`. So please either add it there or remove it here.

Reply:
Removed the 3.8 reference.